### ISSUE TITLE
Expanded images disappear

### ISSUE THREAD

**Issue by catzybluphish** [ranger/ranger#1437]
#### Runtime Environment

- Operating system and version:  Arch
- Terminal emulator and version: urxvt 9.22
- Python version: 3.7.2
- Ranger version/commit: 1.9.2
- Locale: US

#### Current Behavior
Pressing i on an image will not show it expanded. It appears like an empty ranger.
#### Expected Behavior
To view the image expanded
#### Context

I believe there was an issue about this posted here but I couldn't find it so apologies if it's true. I've tried urxvt/urxvt-full with pixbuf support which both showed the image on expansion but it looked choppy, and rendered over the text unlike below image which is perfectly aligned and centered.
![1547060685001](https://user-images.githubusercontent.com/40843060/50974541-2e8eb600-1537-11e9-8677-9d7a0e8115c9.png)
^ is a nice desktop of what I'm trying to achieve
w3m simply won't work even though I asked the owner of this desktop what method he used and he replied w3m IIRC.

Cheers.

**Comment by toonn** [comment]
Problem is the urxvt preview method basically uses a hack to set the terminal's background image. Do you have w3mimgdisplay installed, should be in the w3m package on arch?

**Comment by catzybluphish** [comment]
> Problem is the urxvt preview method basically uses a hack to set the terminal's background image. Do you have w3mimgdisplay installed, should be in the w3m package on arch?

Yes, I am using w3m. I only tried urxvt method because w3m wasn't working (image previews are fine, just expansion doesn't work). Disabling transparency doesn't seem to help either. Would you like to see my configs?

**Comment by toonn** [comment]
Not necessary. Weird that previews work but not when using `i`. How about if you press `C-l` after `i`?

**Comment by catzybluphish** [comment]
> Not necessary. Weird that previews work but not when using `i`. How about if you press `C-l` after `i`?

Nothing. The image expansion feature has never worked for me.

**Comment by catzybluphish** [comment]
Just want to add that the issue persists on st as well.

**Comment by toonn** [comment]
@Vifon, any ideas?

**Comment by Vifon** [comment]
First things first: Are your config files up to date?

**Comment by catzybluphish** [comment]
> First things first: Are your config files up to date?

I regenerated them all and only edited rc.conf to allow image previews. Same result; can preview images but not expand them.

**Comment by aasril** [comment]
It's also not working for me with w3m, tried ueberzug and it works in three different terminal emulators i have installed (gnome-terminal, urxvt, st). Ueberzug also fixes the bug where the preview from previous files persist on preview column.

**Comment by toonn** [comment]
I still suspect this is because urxvt doesn't properly support w3mimgdisplay.
